### ISSUE TITLE
Superheated walls still count towards the limit if they are destroyed.

### ISSUE THREAD

**Issue by Buggy123** [tgstation/tgstation#33610]
Step 1: make 20 superheated walls
Step 2: they're all destroyed anyway.
Step 3: Can't make more superheated walls ever, can't un-superheat them since they're gone.

**Comment by ninjanomnom** [comment]
Limit of what. Always describe the full situation and always give the round id. You need to provide the full context.

**Comment by duncathan** [comment]
Superheated walls?

**Comment by Armhulen** [comment]
clock cult eminence power

**Comment by Buggy123** [comment]
Don't remember the round ID, but I discussed this with the admins who were online at the time. 

I was a clockcult eminence selected from ghosts. The cult did not declare war, they went quiet. When the time came, I helped defend by superheating walls. When the superheated walls were destroyed by various methods, the superheated wall limit, that is, the Eminence's maximum limit of creating 20 superheated walls at a time, was not restored. The expected behavior would be that 1 superheated wall slot was freed per destroyed wall, the actual behavior was that 0 slots were freed in that scenario. Additionally, because the walls were destroyed, it was impossible to un-superheat them manually. Thus it simply became impossible to superheat walls.
